systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/AlternativeReconcileTriggersST.java

```diff
@@ -208,6 +209,18 @@ void testManualTriggeringRollingUpdate() {
      *  7. Try consumer, producer and consume rmeessages again with new certificates
      */
     @Test
+    @Disabled // fix-it ...
```

Comment:
What are the plans for fixing this? Also, any GitHub issues to link to here for this?

Reply:
@tombentley any idea what can be done?

Reply:
Did I disable it? Anyway, I've no recollection of doing this.

Reply:
@tombentley no, I disabled it ... but this is what you had to say about it :-)

``
    @Disabled // fix-it ...
    /*
     * I'm guessing that this test passed before because we never had a consumer in this test,
     * so there was no __consumer_offsets for it to fail on.
     * The test topic which is used has min.isr == replicas,
     * so KafkaAvailability would always have ignored that because it's unrollable otherwise.
     * Now there's a partition with a minisr set to 1 but with >1 replicas,
     * so KafkaAvailability cannot ignore it.
     * And because the key was changed it means that the other brokers don't trust the new broker 0 cert
     * (because it's signed with a key they don't yet trust). So they can't connect to do follower fetches.
     * So we've kind of deadlocked.
     */
``

Reply:
Thanks, I remember now. The test isn't so realistic, having a topic with minisr== replicas. Having 3 replicas and min.isr=2 would be more realistic. So I think we should change the test. WDYT @Frawless @scholzj ?

Reply:
@alesj I think the explanation makes sense. But the `// fix-it ...` comment alone is not the right solution. We should either fix it in this PR (whatever we decide is the right fix - maybe we just remove it, maybe we want to add some additional logging to make the situation more readable for users etc.). Or we should have a Github issue for fixing this later and have it linked here so that we get back to it.

Reply:
@tombentley I think the problem is that having the test more realistic also means it does not pass. When the CA secret is deleted, the operator just generates a new one and rolls the pods. But the cluster has to get partitioned during that. The old (not-yet-rolled) pods and the new (already-rolled) pods will each use distinct set of SSL keys and will form two separate partitions which will not link. So with some traffic going on, this always has to fail because you cannot keep the replicas in-sync. So I think this test will be deleted sooner or later. But we should probably think about if we need to somehow improve the logging in these situations etc. Because it could be confusing for the user.

Reply:
https://github.com/strimzi/strimzi-kafka-operator/issues/4187